connectors/github-outbound/README.md

```diff
@@ -0,0 +1,4 @@
+# GitHub Connector
+
+Connector to create issues, repositories, comments, releases on GitHub. This connector is based on 
+outbound connectors.
```

Comment:
Do you think it makes sense to make a reference to the http-json connector here? E.g.:

> This Connector reuses the base implementation of [HTTP JSON Connector](../http-json) by providing a compatible element template.

Reply:
Absolutely! Nicely phrased. I copy pasted it :D 👍